spec/features/user_can_sign_out_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'feature_helper'
+
+feature 'User can sign In' do
+
+  given(:user) { create(:user) }
+
+  scenario 'registered user try to signed' do
+    sign_in(user)
+
+    expect(page).to have_content 'Sign Out'
+    click_on 'Sign Out'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.